extensions/kind/src/create-cluster.ts

```diff
@@ -174,16 +182,14 @@ export async function createCluster(
       errorMessage = error;
     }
 
-    telemetryLogger.logError('createCluster', {
-      provider,
-      httpHostPort,
-      httpsHostPort,
-      ingressController,
-      error: errorMessage,
-      stdErr: errorMessage,
-    });
+    telemetryOptions.error = errorMessage;
+
```

Comment:
in compose, we're logging the error object (that will include exit code, etc)
https://github.com/containers/podman-desktop/pull/4407/files

here just the message. is it wanted ?